app/src/main/java/com/mk/tasky/core/util/ResultExtension.kt

```diff
@@ -0,0 +1,19 @@
+package com.mk.tasky.core.util
```

Comment:
Mini recommendation: I'd put this in data -> util instead since it reveals implementation details (you're using Retrofit)